qiskit/qobj/qasm_qobj.py

```diff
@@ -313,70 +325,124 @@ def __eq__(self, other):
         return False
 
 
-class QobjDictField(SimpleNamespace):
-    """A class used to represent a dictionary field in Qobj
+class QasmQobjExperimentHeader(QobjDictField):
+    """A header for a single QASM experiment in the qobj."""
+    pass
+
+
+class QasmQobjExperimentConfig(QobjDictField):
+    """Configuration for a single QASM experiment in the qobj."""
+
+    def __init__(self, calibrations=None, **kwargs):
```

Comment:
I don't think there will be a problem here because its all kwargs, but have you tested deepcopy/pickle on an object to make sure everything ends up on the other side of serialization. I just get jumpy around `SimpleNamespace` subclasses now after having to debug it a lot.

Reply:
I just tried this and had no issues :)